resources/run_adl.sh

```diff
@@ -168,34 +173,45 @@ check_mongohoused
 if [[ $? -ne 0 ]]; then
   if [ $ARG = $START ]; then
     echo "Starting $MONGOHOUSED"
-    # Install and start mongohoused
-    git config --global url.git@github.com:.insteadOf https://github.com/
-    # Clone the mongohouse repo
-    if [ ! -d "$MONGOHOUSE_DIR" ]; then
-      git clone $MONGOHOUSE_URI
+    if [[ $HAVE_LOCAL_MONGOHOUSE -eq 1 ]]; then
+        if [ ! -d "$LOCAL_MONGOHOUSE_DIR" ]; then
+            echo "ERROR: LOCAL_MONGOHOUSE_DIR is not a directory"
```

Comment:
Let's print the value of the directory here
```suggestion
            echo "ERROR: $LOCAL_MONGOHOUSE_DIR is not a directory"
```